.github/workflows/pr-reminder.yml

```
@@ -0,0 +1,220 @@
+            });
+            
+            // 리뷰가 없으면 true 반환
+            // 결과는 steps.check_reviews.outputs.result에 저장됨
+            return reviews.length === 0;
+
+      # 세번째 단계 : 디코 알림
+      - name: Send Discord notification
+        # 위에서 저장한 값(리뷰 있는지 없는지?) 확인
+        # 만약 리뷰가 없으면, 디스코드 웹훅 호출하여 리마인더 전송
+        if: steps.check_reviews.outputs.result == 'true'
+        run: |
```

actionlint (1.7.10)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)